metrics/go_metrics.go

```diff
@@ -215,13 +238,25 @@ func NewBlankMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderNa
 	return newMetrics
 }
 
+func getModuleNames(ms map[string][]string) []string {
+	mn := make([]string, len(ms))
+
+	i := 0
+	for n := range ms {
+		mn[i] = n
+		i++
+	}
+
+	return mn
+}
```

Comment:
I recommend changing the `mn` variable to `names` for improved readability.

Reply:
Yeah I agree with the need to change the variable names here